test/testdata/app-scenarios/petset.yaml

```diff
@@ -0,0 +1,496 @@
+        readOnly: true
+    dnsPolicy: ClusterFirst
+    host: localhost.localdomain
+    imagePullSecrets:
+    - name: default-dockercfg-pzhsj
+    nodeName: localhost.localdomain
+    restartPolicy: Always
+    securityContext:
+      seLinuxOptions:
+        level: s0:c5,c0
+    serviceAccount: default
+    serviceAccountName: default
+    terminationGracePeriodSeconds: 30
+    volumes:
+    - name: datadir
+      persistentVolumeClaim:
+        claimName: datadir-mysql-0
+    - emptyDir: {}
+      name: config
+    - emptyDir: {}
+      name: workdir
+    - name: default-token-au2xq
+      secret:
+        secretName: default-token-au2xq
+  status:
+    conditions:
+    - lastProbeTime: null
+      lastTransitionTime: 2016-07-27T02:41:15Z
+      status: "True"
+      type: Initialized
+    - lastProbeTime: null
+      lastTransitionTime: 2016-07-27T03:00:47Z
+      status: "True"
+      type: Ready
+    - lastProbeTime: null
+      lastTransitionTime: 2016-07-27T02:41:09Z
+      status: "True"
+      type: PodScheduled
+    containerStatuses:
+    - containerID: docker://f2406b0f697c525df44b64aec6b1f6024ab88d9df80256426247dc6e9a92cb30
+      image: erkules/galera:basic
+      imageID: docker://sha256:b4780e247a38c12612f539ce1ac8e0988e1781d56fddf719c80fb8d4d7b8bbde
+      lastState: {}
+      name: mysql
+      ready: true
+      restartCount: 0
+      state:
+        running:
+          startedAt: 2016-07-27T02:41:16Z
+    hostIP: 10.0.2.15
+    phase: Running
+    podIP: 172.17.0.2
+    startTime: 2016-07-27T02:41:09Z
+- apiVersion: v1
+  kind: Pod
+  metadata:
+    annotations:
+      kubernetes.io/created-by: |
+        {"kind":"SerializedReference","apiVersion":"v1","reference":{"kind":"PetSet","namespace":"default","name":"mysql","uid":"3900c985-4f5b-11e6-b8a1-080027242396","apiVersion":"apps","resourceVersion":"6790"}}
+      openshift.io/scc: anyuid
+      pod.alpha.kubernetes.io/init-container-statuses: '[{"name":"install","state":{"terminated":{"exitCode":0,"reason":"Completed","startedAt":"2016-07-27T02:41:42Z","finishedAt":"2016-07-27T02:41:42Z","containerID":"docker://2538c65f65557955c02745ef4021181cf322c8dc0db62144dd1e1f8ea9f7fa54"}},"lastState":{},"ready":true,"restartCount":0,"image":"gcr.io/google_containers/galera-install:0.1","imageID":"docker://sha256:56ef857005d0ce479f2db0e4ee0ece05e0766ebfa7e79e27e1513915262a18ec","containerID":"docker://2538c65f65557955c02745ef4021181cf322c8dc0db62144dd1e1f8ea9f7fa54"},{"name":"bootstrap","state":{"terminated":{"exitCode":0,"reason":"Completed","startedAt":"2016-07-27T02:41:44Z","finishedAt":"2016-07-27T02:41:45Z","containerID":"docker://4df7188d37033c182e675d45179941766bd1e6a013469038f43fa3fecc2cc06d"}},"lastState":{},"ready":true,"restartCount":0,"image":"debian:jessie","imageID":"docker://sha256:1b088884749bd93867ddb48ff404d4bbff09a17af8d95bc863efa5d133f87b78","containerID":"docker://4df7188d37033c182e675d45179941766bd1e6a013469038f43fa3fecc2cc06d"}]'
```

Comment:
In this case the describer uses it and since there is no kubelet running we
don't get the "official" output without it.

On Thu, Aug 4, 2016 at 2:48 PM, David Eads <notifications@github.com> wrote:

> In test/testdata/app-scenarios/petset.yaml
> <https://github.com/openshift/origin/pull/9972#discussion_r73580857>:
>
> > +      restartCount: 0
> > +      state:
> > +        running:
> > +          startedAt: 2016-07-27T02:41:16Z
> > +    hostIP: 10.0.2.15
> > +    phase: Running
> > +    podIP: 172.17.0.2
> > +    startTime: 2016-07-27T02:41:09Z
> > +- apiVersion: v1
> > +  kind: Pod
> > +  metadata:
> > +    annotations:
> > +      kubernetes.io/created-by: |
> > +        {"kind":"SerializedReference","apiVersion":"v1","reference":{"kind":"PetSet","namespace":"default","name":"mysql","uid":"3900c985-4f5b-11e6-b8a1-080027242396","apiVersion":"apps","resourceVersion":"6790"}}
> > +      openshift.io/scc: anyuid
> > +      pod.alpha.kubernetes.io/init-container-statuses: '[{"name":"install","state":{"terminated":{"exitCode":0,"reason":"Completed","startedAt":"2016-07-27T02:41:42Z","finishedAt":"2016-07-27T02:41:42Z","containerID":"docker://2538c65f65557955c02745ef4021181cf322c8dc0db62144dd1e1f8ea9f7fa54"}},"lastState":{},"ready":true,"restartCount":0,"image":"gcr.io/google_containers/galera-install:0.1","imageID":"docker://sha256:56ef857005d0ce479f2db0e4ee0ece05e0766ebfa7e79e27e1513915262a18ec","containerID":"docker://2538c65f65557955c02745ef4021181cf322c8dc0db62144dd1e1f8ea9f7fa54"},{"name":"bootstrap","state":{"terminated":{"exitCode":0,"reason":"Completed","startedAt":"2016-07-27T02:41:44Z","finishedAt":"2016-07-27T02:41:45Z","containerID":"docker://4df7188d37033c182e675d45179941766bd1e6a013469038f43fa3fecc2cc06d"}},"lastState":{},"ready":true,"restartCount":0,"image":"debian:jessie","imageID":"docker://sha256:1b088884749bd93867ddb48ff404d4bbff09a17af8d95bc863efa5d133f87b78","containerID":"docker://4df7188d37033c182e675d45179941766bd1e6a013469038f43fa3fecc2cc06d"}]'
>
> seems weird that our test data would have statuses.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9972/files/fdbb46f05fe687c7d2791edf8f685f181e355e91#r73580857>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p6u3uQME8Okm2-7Sn028Ex3IE85_ks5qcjQZgaJpZM4JRYA_>
> .
>